content/dota_addons/legion_td/panorama/layout/custom_game/team_select_team.xml

```diff
@@ -1,24 +1,22 @@
 <root>
-    <styles>
-        <include src="file://{resources}/styles/dotastyles.css" />
-        <include src="file://{resources}/styles/custom_game/team_select.css" />
-    </styles>
-    <scripts>
-        <include src="file://{resources}/scripts/custom_game/team_select_team.js" />
-    </scripts>
-    <!-- <Panel class="TeamSelectTeamWrapper"> -->
-    <Button class="TeamSelectTeam" onactivate="OnJoinTeamPressed()">
-        <Panel id="TeamBackgroundGradient" />
-        <Panel id="TeamBackgroundGradientHighlight" />
-        <Panel id="TeamGroup">
-            <Panel id="TeamHeader">
-                <Panel id="TeamLogo" />
-                <Label id="TeamNameLabel" text="TEAM_NAME" />
-                <Label id="JoinTeamLabel" text="#custom_game_team_select_join" />
-            </Panel>
-            <Panel id="TeamPlayerDivider" />
-            <Panel id="PlayerList" />
-        </Panel>
-    </Button>
-    <!-- </Panel> -->
-</root>
+	<styles>
+		<include src="file://{resources}/styles/dotastyles.css" />
+		<include src="file://{resources}/styles/custom_game/team_select.css" />
+	</styles>
+	<scripts>
+		<include src="file://{resources}/scripts/custom_game/team_select_team.js" />
+	</scripts>
+	<Button class="TeamSelectTeam" onactivate="OnJoinTeamPressed()">
+		<Panel id="TeamBackgroundGradient" />
+		<Panel id="TeamBackgroundGradientHighlight" />
+		<Panel id="TeamGroup" >
+			<Panel id="TeamHeader">
+				<Panel id="TeamLogo" />
+				<Label id="TeamNameLabel" text="TEAM_NAME" />
+				<Label id="JoinTeamLabel" text="#custom_game_team_select_join" />
+			</Panel>
+			<Panel id="TeamPlayerDivider" />
+			<Panel id="PlayerList" />
+		</Panel>
+	</Button>
+</root>
```

Comment:
Could you make this to fill the whole container? Atm, there is a margin at the top and bottom that would look better if removed.
![image](https://user-images.githubusercontent.com/9084670/72279755-a7619b80-35eb-11ea-876e-48f9a253e560.png)

Reply:
Also please have an empty line at the end of each file